flags/mod.ts

```diff
@@ -2,6 +2,8 @@
 // This module is browser compatible.
 
 /**
+ * @deprecated (will be removed after 1.0.0) Import from `"https://deno.land/std@$STD_VERSION/cli/flags.ts` instead.
```

Comment:
Perhaps, we're okay to remove after 0.208.0. @kt3k, WDYT?

Reply:
`std/flags` exists for long time and is heavily used. I think we shouldn't remove this that soon

Reply:
Is `1.0.0` too late or what do you think about the timing?

Reply:
I think removal at 1.0.0 is safer and more appropriate regarding the usage of this module.